src/config.js

```diff
@@ -60,6 +66,9 @@ const SS58_ADDR_TYPE = {
   REEF: 42,
   ACALA: 10,
   ELIXXIR: 55,
+  ALEPHZERO: 42,
```

Comment:
This will require contacting the respective teams. 
42 is the default/testnet prefix.